core/src/main/java/com/predic8/membrane/core/util/ExceptionUtil.java

```diff
@@ -25,4 +26,14 @@ public static String concatMessageAndCauseMessages(Throwable throwable) {
         } while (throwable != null);
         return sb.toString();
     }
+
+    public static Throwable getRootCause(Throwable t) {
```

Comment:
why do we throw the outer exceptions away? usually, they are there to provide context.